src/components/FavouriteListItem/FavouriteListItem.jsx

```diff
@@ -27,7 +29,9 @@ static propTypes = {
                     <div className="favourite-list-item__title">{item.name}</div>
                     <div className="favourite-list-item__tagline">{item.tagline}</div>
                     <div className="favourite-list-item__description">{item.description}</div>
-                    <button type="button" className="favourite-list-item__button">Open</button>
+                    <Link to={`/details/${item.id}`}>
+                        <button type="button" className="favourite-list-item__button">Open</button>
+                    </Link>
```

Comment:
Don't you want to create a separate component, since you have similar logic in separate places?